driver/driver.go

```diff
@@ -371,6 +372,10 @@ func (c *Conn) Prepare(query string) (driver.Stmt, error) {
 
 // ExecContext is an optional interface that may be implemented by a Conn.
 func (c *Conn) ExecContext(ctx context.Context, query string, args []driver.NamedValue) (driver.Result, error) {
+	if len(args) > 255 {
```

Comment:
Can you change this to a constant somewhere so its easy to change when the underlying limitation is fixed?